horton/gbasis/gobasis.py

```diff
@@ -238,6 +247,21 @@ def load(self):
         self._to_segmented()
         self._normalize_contractions()
 
+    def dump(self, filename=None):
+        """ Dumps the basis set in the gbs format
+        
+        Parameter
+        ---------
+        filename : str
+            Name of the gbs file that will be created
+            By default, the original filename is used with '_mod' appended to the end of
+            the filename (before extension)
+        """
+        self.load()
+        if filename is None:
+            filename = ''.join(self.filename.split('.')[:-1] + ['_mod.gbs'])
+        load_basis_atom_map_gbs(filename)
```

Comment:
Should be load instead of dump, I guess. Could you also turn it into

    if filename.endswith('.gbs'):
        dump_basis_atom_maps_gbs(filename)
    else:
        raise IOError('File format not supported: %s' % filename)